README.md

```diff
@@ -3,3 +3,56 @@
 # github-exporter
 
 The github-exporter exports Prometheus metrics for Github data.
+
+
+
+### Example Execution
+
+```
+./github-exporter daemon --service.collector.issue.customlabels='[ "kind/okr,goal/achieved", "kind/okr,goal/missed", "postmortem,team/batman", "postmortem,team/magic", "postmortem,team/spirit" ]' --service.github.auth.token=$(cat ~/.credential/github-exporter-github-token)
+```
+
+
+
+### Example Queries
+
+Showing a graph of the total number of open and closed issues.
+
+```
+github_exporter_issue_states_count
+```
+
+Showing a graph of open and closed postmortem issues.
+
+```
+github_exporter_issue_label_count{label="postmortem"}
+```
+
+Showing a graph of open and closed bug issues.
+
+```
+github_exporter_issue_label_count{label="kind/bug"}
+```
+
+Showing a graph of open and closed postmortem issues per team.
+
+```
+github_exporter_issue_labels_count{labels="postmortem,team/batman"}
+github_exporter_issue_labels_count{labels="postmortem,team/magic"}
+github_exporter_issue_labels_count{labels="postmortem,team/spirit"}
+```
+
+```
+github_exporter_issue_labels_count{labels=~"postmortem,team/.*"}
```

Comment:
I wonder if it makes sense to extract labels for known labels. E.g. `{postmortem="true",team="spirit",labels="unhandled_label"}`. This would make those queries more reliable.

Reply:
I see what you mean and I agree with your idea. This is hackathon though and I do not want to spend time on making things 100% now. I want to get shit done. I think the implementation is also way more complicated then because you need to construct the gauge descriptions at runtime. Our current approach is to have them defined statically. @giantswarm/sig-monitoring should try to come up with a guideline for the future so we can make it how it should be.

Reply:
> I think the implementation is also way more complicated then because you need to construct the gauge descriptions at runtime

Not really I thought on predefined stuff. The rest would still be in `labels`.

Reply:
When it is predefined you cannot configure it but have to code it. I found the dynamic solution useful as a starter to be able to more easily play around with different queries without the need to code. When we later come to conclusions we can hard code whatever we permanently need.